src/main/kotlin/no/nav/persondataapi/konfigurasjon/RetryPolicy.kt

```diff
@@ -0,0 +1,72 @@
+package no.nav.persondataapi.konfigurasjon
+
+import io.netty.handler.timeout.ReadTimeoutException
+import io.netty.handler.timeout.WriteTimeoutException
+import org.slf4j.LoggerFactory
+import reactor.util.retry.Retry
+import java.net.ConnectException
+import java.time.Duration
+import java.util.concurrent.TimeoutException
+import kotlinx.coroutines.delay
+
+object RetryPolicy {
+
+    private val log = LoggerFactory.getLogger(RetryPolicy::class.java)
+
+    /** Hvilke exceptions som regnes som retrybare */
+    val retryableExceptions: (Throwable) -> Boolean = { t ->
+        t is TimeoutException ||
+                t is ReadTimeoutException ||
+                t is WriteTimeoutException ||
+                t is ConnectException ||
+                t.message?.contains("Serverfeil") == true
+    }
+
+    /** Reactor Retry for WebClient */
+    fun reactorRetrySpec(
+        attempts: Long = 3,
+        kilde:String = "underliggende system",
```

Comment:
Nitpick
```suggestion
        kilde: String = "Underliggende system",
```

(og burde ikke dette være en påkrevd ting?)